myosuite/agents/baselines_DEPRL/myoLegWalk_20230514/myoLeg/checkpoints/read_for_checkpoint.txt

```diff
@@ -0,0 +1 @@
+If you are looking for a policy checkpoint, they are too large to be saved here. Open the README file from the agents/ folder and you'll see how you can download one.
```

Comment:
can you add a link to the README you are referring to? it will be easier to find it :)